src/main/java/edu/usf/cutr/tba/manager/TravelBehaviorDataAnalysisManager.java

```diff
@@ -56,9 +56,12 @@
     public TravelBehaviorDataAnalysisManager() throws FirebaseFileNotInitializedException {
         mFirebaseReader = new FirebaseReader();
         mCSVFileWriter = new CSVFileWriter();
-        mKmlFileWriter = new KmlFileWriter();
         mOneDayTravelBehaviorRecordList = new ArrayList<>();
         mProgramOptions = ProgramOptions.getInstance();
+
+        if (!mProgramOptions.getNoKMZ()) {
```

Comment:
I think it would be easier to read this code if we call the command line parameter `skipKMZ` - that way we don't have to deal with a double-negative. So this would look more like:

```java
if (!mProgramOptions.skipKMZ()) {
```